clang/runtime/dpct-rt/include/kernel.hpp.inc

```diff
@@ -46,10 +46,10 @@
 #include <dlfcn.h>
 #endif
 
-#if (__GNUC__ < 8) && !defined(__llvm__)
-#include <experimental/filesystem>
-#else
+#if defined(__has_include) && __has_include(<filesystem>)
 #include <filesystem>
+#else
+#include <experimental/filesystem>
```

Comment:
Should we support elder version compilers that don't have <experimental/filesystem>?

Reply:
experimental/filesystem was added in gcc 5.3.0 which was released in December 2015.

Reply:
The GCC compiler requirements for building migrated code is:

GCC* 7.5 or later for SUSE, 9.3 or later for Ubuntu, 8.2.1 or later for Red Hat, 12.0.1 or later for Fedora